context/pokemonReducer.tsx

```diff
@@ -0,0 +1,13 @@
+import { SAVE_FAV_POKEMON } from './types';
+
+export default (state: any, action: any) => {
```

Comment:
I'm pretty sure you can easily add types for both `state`, `action`, example here:
https://github.com/piotrwitek/react-redux-typescript-guide/pull/118/files
DM me on Slack if you'd have problems